### PR TITLE
AG-11329 Implement keyboard navigation in map series

### PR DESCRIPTION
**Pull request by olegat**
https://ag-grid.atlassian.net/browse/AG-11329

### CODE REVIEW

File: packages/ag-charts-enterprise/src/series/map-line-background/mapLineBackgroundSeries.ts
Comment (alantreadway) [inline packages/ag-charts-enterprise/src/series/map-line-background/mapLineBackgroundSeries.ts:1]: Background series should not be focusable IMO

File: packages/ag-charts-enterprise/src/series/map-shape-background/mapShapeBackgroundSeries.ts
Comment (alantreadway) [inline packages/ag-charts-enterprise/src/series/map-shape-background/mapShapeBackgroundSeries.ts:1]: Background shape series should not be focusable based upon the JIRA description: https://ag-grid.atlassian.net/browse/AG-11329 ?

Comment (olegat) [reply packages/ag-charts-enterprise/src/series/map-shape-background/mapShapeBackgroundSeries.ts:1]: Done.

File: packages/ag-charts-enterprise/src/series/map-line-background/mapLineBackgroundSeries.ts
Comment (olegat) [reply packages/ag-charts-enterprise/src/series/map-line-background/mapLineBackgroundSeries.ts:1]: Done.

**Review by alantreadway — APPROVED** [review]
LGTM - thanks for the updates!

**Pull request merged**